.ci/install_crio.sh

```diff
@@ -23,7 +23,7 @@ if [ ! -d "$GOBIN" ]
 then
         mkdir -p "$GOBIN"
 fi
-ln -s $(command -v go-md2man) "$GOBIN"
+ln -sf $(command -v go-md2man) "$GOBIN"
```

Comment:
I've since seen a different issue here - if we are running on top of a dirty tree then the `go-md2man` file found from the `command` is already the on in the `$GOBIN`, and then ln complain and fails with 'source file is the same as dest file`.
I've got a local workaround - one of 3 options I think:
- if the dest already exists, skip the `ln`
- if the dest already exists, force remove it before the `ln`
- ignore the error from the `ln` with a `|| true`

@chavafg - any thoughts or preferences from you on this?

(I'm happy to rework the PR btw, but we then run the risk of bumping into the Travis monster, so maybe better as a new PR to update the code)

Reply:
Hi @grahamwhaley, 
Either the 1st or second option would be ok for me. 
I'll merge this one and you can rework on a new one to not hit the travis bug.